api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -203,13 +203,7 @@ def _get_gtfs_feeds(
                 for country_code, subdivision_name, municipality in unique_locations
             ]
 
-            latest_dataset, bounding_box = next(
-                filter(
-                    lambda dataset: dataset[0] is not None and dataset[1] is not None and dataset[0].latest,
-                    zip(datasets, bounding_boxes),
-                ),
-                (None, None),
-            )
+            latest_dataset = next((dataset for dataset in datasets if dataset is not None and dataset.latest), None)
```

Comment:
Just curious: How are we sure only 1 dataset per feed has the latest boolean set to true?